reframe/frontend/reporting/utility.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2016-2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import abc
+import re
+import statistics
+import types
+from collections import namedtuple
+from datetime import datetime, timedelta
+from numbers import Number
+from .storage import StorageBackend
+
+
+class Aggregator:
+    @classmethod
+    def create(cls, name, *args, **kwargs):
```

Comment:
Does it make sense to have a `count` aggregator? What do you think?

Reply:
I gave a second thought to it and I don't think it makes sense at the moment. This feature is all about performance aggregating only the performance values, so counting doesn't make sense imho. We will have options to retrieve full test case and session data as JSON so it can be fed to external tools.